OMMBV/igrf13.f

```diff
@@ -38,12 +38,12 @@ subroutine igrf_step(out,pos,t,date,scalar,dir,height)
       ! when to terminate
       call ecef_to_geodetic(pos, latitude, elong, h)
       ! stop moving position if we go below height
-      if (h.le.(height)) then
-        scalar = 0
-      else if (h.le.(height+10.)) then
+      !if (h.le.(height)) then
```

Comment:
This was put in previously to slow things down when we get near the termination point. I found it improved the variance in the termination height. Does removing this change this termination uncertainty? I believe there is a unit test plot that covers that one.